### ISSUE TITLE
Add Coc.nvim for autocompletion

### ISSUE THREAD

**Issue by jyash8** [AstroNvim/astrocommunity#657]
### Is your feature related to a problem?

No.

### Describe the new feature

COC.nvim plugin provides autocompletion support for a lot of languages.

**Comment by Uzaaft** [comment]
@jyash8  Feel free to open a PR. :)

**Comment by taskylizard** [comment]
I don't think this is feasible, the lsp features are integrated heavily in astronvim core, so swapping that for coc.nvim doesn't sound good.

**Comment by Uzaaft** [comment]
It is possible in fact.

**Comment by luxus** [comment]
looks like the interest is pretty low? should we close this?

**Comment by Uzaaft** [comment]
Let it stay open @luxus
